pkg/lint/rules/values.go

```diff
@@ -60,6 +60,14 @@ func validateValuesFileExistence(valuesPath string) error {
 	return nil
 }
 
+func copyMap(src map[string]interface{}) map[string]interface{} {
```

Comment:
@hickeyma  Does `helm`  have a method for map copy?

Reply:
No.